spotify_player/src/cli/mod.rs

```diff
@@ -56,6 +56,32 @@ pub enum IdOrName {
     Name(String),
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum PlaylistCommand {
+    New {
+        name: String,
+        public: bool,
+        collab: bool,
+        description: String,
+    },
+    Delete {
+        id: PlaylistId<'static>,
+    },
+    List,
+    Import {
+        from: PlaylistId<'static>,
+        to: PlaylistId<'static>,
+        delete: bool,
+    },
+    Fork {
+        id: PlaylistId<'static>,
+    },
+    Update {
```

Comment:
`update` is confusing. This should be `SyncImport` or `UpdateImport`. When I see `update`, I will expect it to update a playlist's data like name, description, etc.

Reply:
Is this in reference to the cli command name too?

Reply:
Yeah, `Sync` works.